src/material-experimental/mdc-button/_button-base.scss

```diff
@@ -40,6 +40,11 @@
   .mdc-button__label {
     z-index: 1;
   }
+
+  // The focus indicator should match the bounds of the entire button.
+  .mat-mdc-focus-indicator {
+    @include mat-fill();
```

Comment:
Before this change, this element had essentially no styles, and thus the focus indicator actually rendered relative to the parent button. Now that we've added `position: relative` to this element, the fact that it has no styles is problematic (the focus indicator doesn't render with the correct bounding box). Consequently, use `mat-fill` to make this element match the bounds of the entire button.